irods/query.py

```diff
@@ -36,7 +36,8 @@
     "SELECT_COUNT": 6,
 }
 
-IRODS_QUERY_LIMIT = os.getenv("IRODS_QUERY_LIMIT", 500)
+IRODS_QUERY_LIMIT = 500
```

Comment:
`IRODS_QUERY_LIMIT` is used on L200. That means the number is always 500 now. Should we replace that with a call to the `irods_query_limit` getter?

Reply:
Ohhh, I see. Okay, let's at least leave a comment saying that the value is changed by that setting

Reply:
done